torch/ao/quantization/experimental/observer.py

```diff
@@ -138,3 +138,5 @@ def forward(self, x_orig):
         self.min_val.copy_(min_val)
```

Comment:
`min_val` is declared as `min_val=torch.Tensor,`. should it instead be declared as `min_val: torch.Tensor,`?

Reply:
also probably if b=0, k=0 (or other reasonable defaults) could be put into the base class constructor defaults, ApOTObserver can do without overriding the constructor at all